src/pyOpenMS/pyopenms/_dataframes.py

```
@@ -1,23 +1,31 @@
+
+And utility functions:
+- common_meta_value_types: Dictionary for numpy type mapping of common meta values
 """
 from __future__ import annotations
 
 __all__ = [
     'peptide_identifications_to_df',
     'update_scores_from_df',
+    'common_meta_value_types',
+    '_add_meta_values',
 ]
```

Ruff (0.14.11)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)